plugins/ad9371.c

```diff
@@ -1656,6 +1713,7 @@ static GtkWidget * ad9371_init(struct osc_plugin *plugin, GtkWidget *notebook, c
 	/* Update all widgets with current values */
 	printf("Updating widgets...\n");
 	update_widgets();
+	sample_frequency_changed_cb(NULL);
```

Comment:
this function calls rx_freq_info_update() which is also called on the next line. So it would get called twice. Maybe you just needed the other call in rx_freq_info_update() which is glb_settings_update_labels(). So call that directly.